app/experimenter/static/js/components/DesignForm.js

```diff
@@ -26,14 +26,22 @@ class DesignForm extends React.PureComponent {
 
   async componentDidMount() {
     const data = await makeApiRequest(this.getEndpointUrl());
+
+    const errors = {
+      variants: data.variants.map(() => ({})),
+    };
+
     this.setState({
       loaded: true,
       data,
+      errors,
     });
   }
 
   addBranch() {
     const { data } = this.state;
+    const { errors } = this.state;
```

Comment:
I think you can do 42 and 43 on the same line?

Reply:
this got changed around.